feature/DOC-769: Document new resolve conversation options for the comments plugin (#1876)
tinymce/tinymce-docs: plugins/premium/comments.md

Follow this procedure to delete a comment. This option is not available for the first comment in a conversation.

1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.
1. Select **Delete** from the menu items.
1. The following options appear in the comments sidebar:<br/>
![delete comment]({{site.baseurl}}/images/comments-delete-comment.png)
1. Click **Cancel** to cancel the action or **Delete** to remove the comment from the conversation.

### Delete conversation

This option is only available for the first comment in a conversation. Once the comment is saved, follow this procedure to delete a conversation.

1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.
1. Select **Delete conversation** from the menu items.
1. The following decision dialog box will appear:<br/>
![delete conversation]({{site.baseurl}}/images/comments-delete-conversation.png)
1. Click **Cancel** to cancel the action or **Delete** to remove the conversation.

**Result**: The conversation and all its subsequent comments will be deleted.

### Resolve conversation

> 

Reviewer: ```suggestion

{{site.requires_5_8v}}

```

This option is only available for the first comment in a conversation. Once a comment is saved, follow this procedure to resolve a conversation.

> 1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.

Reviewer: Looking at: https://www.tiny.cloud/docs/configure/contributing-docs/#talkingabouttheuserinterface
We would use 'select' per https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui
however this is out of scope for this PR. It would lead to replacing a lot of 'clicks'.
also, comments isn't supported on mobile (which may only be related to the sidebar size): https://www.tiny.cloud/docs/mobile/

> 1. Click on this ![3 dots]({{site.baseurl}}/images/comments-3dots.png) icon above the comments box to expand the menu.

Reviewer: ```suggestion
1. Click on the ellipsis ![(ellipsis - 3 horizontal dots)]({{site.baseurl}}/images/icons/more-drawer.svg) icon above the comment box to expand the menu.
```

1. Select **Resolve conversation** from the menu items.
1. The following decision dialog box will appear:<br/>
![resolve conversation]({{site.baseurl}}/images/comments-resolve-conversation.png)

> 1. Click **Cancel** to save or **Resolve** to resolve the conversation.

Reviewer 2: "Cancel to save" seems wrong?

Author: Yup that's wrong, reworded.

> **Result**: The conversation be resolved.

Reviewer 2: ```suggestion
**Result**: The conversation will be resolved.
```

1. Click on the ellipsis ![(ellipsis - 3 horizontal dots)]({{site.baseurl}}/images/icons/image-options.svg) icon above the comments box to expand the menu.
1. Select **Delete** from the menu items.
1. The following options appear in the comments sidebar:<br/>
![delete comment]({{site.baseurl}}/images/comments-delete-comment.png)
1. Click **Cancel** to cancel the action or **Delete** to remove the comment from the conversation.

### Delete conversation

This option is only available for the first comment in a conversation. Once the comment is saved, follow this procedure to delete a conversation.

1. Click on the ellipsis ![(ellipsis - 3 horizontal dots)]({{site.baseurl}}/images/icons/image-options.svg) icon above the comments box to expand the menu.
1. Select **Delete conversation** from the menu items.
1. The following decision dialog box will appear:<br/>
![delete conversation]({{site.baseurl}}/images/comments-delete-conversation.png)
1. Click **Cancel** to cancel the action or **Delete** to remove the conversation.

**Result**: The conversation and all its subsequent comments will be deleted.

> ### Resolve conversation

Author: I added a note mentioning that these settings are required for this feature.

});
```

##### `tinycomments_can_resolve`

> 

Reviewer: ```suggestion

{{site.requires_5_8v}}

```

> _Optional_ - By adding this option, a _Resolve Conversation_ item will be added to the dropdown menu of the first comment in a conversation. This callback sets the author permissions for _resolving comment conversations_.

Reviewer: ```suggestion
_Optional_ - This option adds a _Resolve Conversation_ item to the dropdown menu of the first comment in a conversation. This callback sets the author permissions for _resolving comment conversations_.
```
